modules/installation-configuration-parameters.adoc

```diff
@@ -927,10 +927,14 @@ Additional Azure configuration parameters are described in the following table:
 |====
 |Parameter|Description|Values
 
-|`controlPlane.platform.azure.osDisk.diskSizeGB`
+|`controlPlane.platform.azure.osDisk.diskSizeGB`, `compute.platform.azure.osDisk.diskSizeGB`, `platform.azure.osDisk.diskSizeGB`
 |The Azure disk size for the VM.
 |Integer that represents the size of the disk in GB. The minimum supported disk size is `1024`.
 
+|`controlPlane.platform.azure.osDisk.diskType`, `compute.platform.azure.osDisk.diskType`, `platform.azure.osDisk.diskType`
+|`diskType` defines the type of disk.
+|`standard_LRS`, `premium_LRS`, `standardSSD_LRS`. For control panel nodes: `premium_LRS`. `standardSSD_LRS`. The default is `premium_LRS`.
```

Comment:
I'm a little confused by this description. I see the 3 values, first. But then it says for control plane nodes and gives only 2 of the values (at least I think it's meant to, there is a period in between the two, but maybe it was meant to be a comma?). Does that mean that all 3 are applicable for just compute.* and platform.*, but only 2 are applicable for controlPlane.*? And the default for all is premium? I'd just try to make this a bit clearer

Reply:
Yeah I the period was meant to be a comma, I see what you mean as well. I think I'll separate them to make the values more clear. Thank you!